### PR TITLE
Zookeeper fix

### PR DESCRIPTION
**Pull request by fschueler**
This should fix the issue with isRunning() and the myid files that should be created in the data-directory of each zookeeper server.

### CODE REVIEW

File: peel-extensions/src/main/scala/org/peelframework/zookeeper/beans/system/Zookeeper.scala [hunk 1]
```diff
@@ -59,7 +59,14 @@ class Zookeeper(
   private def start(s: Zookeeper.Server) = {
     logger.info(s"Starting zookeeper at ${s.host}:${s.leaderPort}:${s.quorumPort}")
     val user = config.getString(s"system.$configKey.user")
-    shell ! s""" ssh $user@${s.host} ${config.getString(s"system.$configKey.path.home")}/bin/zkServer.sh start """
+    shell !
+      s"""
+        |ssh -t -t "$user@${s.host}" << SSHEND
+        |  ${config.getString(s"system.$configKey.path.home")}/bin/zkServer.sh start
+        |  echo ${s.id} > ${config.getString(s"system.$configKey.config.dataDir")}/myid
```
Comment (aalexandrov) [inline peel-extensions/src/main/scala/org/peelframework/zookeeper/beans/system/Zookeeper.scala]: It will be better if we have a dedicated variable for the PID. We had issues before when placing the PID file in the shared folder if the same installation is started simultaneously from multiple hosts (see #57) and the associated fix.

Comment (fschueler) [inline peel-extensions/src/main/scala/org/peelframework/zookeeper/beans/system/Zookeeper.scala]: Oh, I see. I actually set the dataDir to a local folder on each host so there was no problem. But as it turns out I also misunderstood how zookeeper is supposed to work a little bit and assumed that it is started on every node. Does a shared dataDir and dedicated folder for PID and myid files make sense?

Comment (aalexandrov) [inline peel-extensions/src/main/scala/org/peelframework/zookeeper/beans/system/Zookeeper.scala]: OK, I think I am missing some context and interpreting your changes wrong. First I thought that the `myid` file keeps track of the PID of the Zookeeper server but it seems that you are just writing out the server ID (e.g. the suffix `XXX` of the `server.XXX` string). Why is this necessary?

**Comment by aalexandrov** [comment]
OK, I found my answer [in the Zookeeper documentation](https://zookeeper.apache.org/doc/r3.4.5/zookeeperAdmin.html) which explains why the `myid` file is needed.

Looks good to merge.
